src/az/ingress/patterns/singleton/Path.java

```diff
@@ -0,0 +1,13 @@
+package az.ingress.patterns.singleton;
+
+public final class Path {
```

Comment:
why Path class is final?

Reply:
There is no point in declaring the class final here. This is my mistake. Thanks for your feedback.